wave_docs/api.mdx

```diff
@@ -86,6 +87,189 @@ The endpoint returns the name of the container request made available by Wave.
 | `targetImage`    | The Wave container image name e.g. `wave.seqera.io/wt/0123456789/library/ubuntu:latest`. |
 | `expiration`     | The expiration timestamp of the Wave container using ISO-8601 format.                    |
 
+## POST `/v1alpha2/container`
+
+This endpoint allows you to submit a request to access a private container repository via Wave, or build a container image on-the-fly with a Dockerfile or Conda recipe file.
+
+The endpoint returns the name of the container request made available by Wave.
+
+### Request body
+
+```json
+{
+    containerImage: string,
+    containerFile: string,
+    containerConfig: {
+        entrypoint: string[],
+        cmd: string[],
+        env: string[],
+        workingDir: string
+        layers: [
+            {
+                location: string,
+                gzipDigest: string,
+                gzipSize: string,
+                tarDigest: string
+            },
+            ...
+        ]
+    },
+    condaFile: string,
+    spackFile: string,
+    containerPlatform: string,
+    buildRepository: string,
+    cacheRepository: string,
+    timestamp: string,
+    fingerprint: string,
+    freeze: boolean,
+    towerAccessToken: string,
+    towerRefreshToken: string,
+    towerEndpoint: string,
+    towerWorkspaceId: number,
+    packages: {
+                  type: string,
+                  packages: string[],
+                  channels: string[],
+                  envFile: string,
+                  condaOpts: {
+                                mambaImage: string,
+                                commands: string[],
+                                basePackages: string
+                              }
+                  spackOpts:{
+                                commands: string[],
+                                basePackages: string
+                            }                     
+                  
+              }
+}
+```
+
+Note: You can read the description of al attributes except packages from [here](#container-token-request-attributes)
```

Comment:
Admonition?

all* attributes, except packages from...

Reply:
@munishchouhan, what are the container token request attributes? Thanks!